api/samples/memval_simple.c

```diff
@@ -403,7 +421,7 @@ event_exit(void)
 DR_EXPORT void
 dr_client_main(client_id_t id, int argc, const char *argv[])
 {
-    drreg_options_t ops = {sizeof(ops), 3 /*max slots needed*/, false};
+    drreg_options_t ops = {sizeof(ops), 3 /*max slots needed*/, true};
```

Comment:
Gotcha, makes sense. I'll run some quick tests to make sure everything still works, but it looks like the conservative setting is *not* needed, as you said.